packages/flutter/lib/src/cupertino/desktop_text_selection.dart

```diff
@@ -49,7 +49,7 @@ class _CupertinoDesktopTextSelectionControls extends TextSelectionControls {
     Offset selectionMidpoint,
     List<TextSelectionPoint> endpoints,
     TextSelectionDelegate delegate,
-    ClipboardStatusNotifier? clipboardStatus,
+    ValueListenable<ClipboardStatus>? clipboardStatus,
```

Comment:
Is the idea here just to simplify things since no methods from ClipboardStatusNotifier are needed here?

Reply:
Nevermind the answer is yes.  I answered my own question by reading the original PR https://github.com/flutter/flutter/pull/98951.